scripts/ci/fetch.github.sh

```diff
@@ -23,7 +23,7 @@ set -e
 
 # install dependencies
 sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 4B7C549A058F8B6B
-echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/4.2 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb.list
+echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/5.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb.list
```

Comment:
Use the right Unbuntu version for this package. 
In the github actions we use we are using [ubuntu-latest](https://github.com/actions/runner-images?tab=readme-ov-file#available-images) as version and from the github runner page it is ubuntu 22.04 and as code name "Jammy Jellyfis" short "jammy".

```suggestion
echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu jammy/mongodb-org/5.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb.list
```